src/System.IO.FileSystem/tests/Resources/System.IO.FileSystem.Tests.rd.xml

```diff
@@ -0,0 +1,6 @@
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library>
+    <!-- Needed because of FileOptions instance in [Theory] data which causes xunit to reflect on its ToString() -->
```

Comment:
Although since I now just realized that this rd.xml is in tests, so perhaps it's not critical to remove the workaround anyway...